fetch.bs

```diff
@@ -3133,6 +3133,32 @@ or an <a>implementation-defined</a> value.
 </div>
 
 
+<h3 id=storage-keys-for-requests>Storage keys for requests</h3>
+
+<p class=note>Requests made to Blob URLs (other than those corresponding to navigations) are
+prevented from succeeding if the <a spec=storage>storage key</a> of the
+<a>environment settings object</a> making the request is different than the
+<a spec=storage>storage key</a> of the <a>environment settings object</a> corresponding to where
+the Blob URL was created.
```

Comment:
This feels like it would work better after line 5028, instead of near this algorithm.

Reply:
I was thinking this would make sense in the Infrastructure section like how the `HTTP cache partitioning` section and `determine the HTTP cache partition` algorithm are, but I've moved the algorithm under the `Scheme fetch` section now (and removed the note since it seems kinda out of place now. WDYT?